ftplugin/latex-suite/outline.py

```diff
@@ -100,9 +105,11 @@ def getSectionLabels_Root(lineinfo, section_prefix, label_prefix):
             # :          e^{i\pi} + 1 = 0
             #
             # Use the current "section depth" for the leading indentation.
-            print >>outstr, '>%s%s\t\t<%s>' % (' ' * (2 * pres_depth + 2),
+            print('>%s%s\t\t<%s>' % (' ' * (2 * pres_depth + 2),
```

Comment:
Could you please use outsr.write() here or is there something that I miss why print() would be a better choice? IMHO using outstr.write() makes it more obvious that this is not written to stdout. Do not forget to add the newline character.